apps/package.json

```diff
@@ -141,6 +142,7 @@
     "karma-webpack": "^4.0.2",
     "lazysizes": "^4.0.0-rc1",
     "load-grunt-tasks": "3.5.0",
+    "loadable-components": "2.2.3",
```

Comment:
![image](https://user-images.githubusercontent.com/1615761/66659409-4dba9100-ebf8-11e9-9d90-eaf0551dbf14.png)

It looks like this is an older/unsupported version of this library.  It's unclear to me what the difference between the old a new one is since they're pointing to the same GitHub repo.  Did you have a particular reason for picking this one?

Reply:
Great feedback Brad! 

The older version of `loadable-components` is because we aren't on Babel 7 yet.